docs/io/fits/appendix/history.rst

```diff
@@ -4,8 +4,8 @@ astropy.io.fits History
 ***********************
 
 Prior to its inclusion in Astropy, the `astropy.io.fits` package was a stand-
-alone package called `PyFITS`_.  Though for the time being active development
-is continuing on PyFITS, that development is also being merged into Astropy.
+alone package called `PyFITS`_.  PyFITS is no longer actively maintained, and
```

Comment:
Just remove the `PyFITS is .. in Astropy` sentence altogether?

Reply:
I disagree a bit here because I still get help calls about PyFITS and have to keep telling people to switch to `astropy.io.fits`. I would like to keep this sentence so it shows up on the Internet and let people know.

Reply:
OK, I take actual experience any time over uninformed opinion. Just keep it!